src/main/java/com/kainos/ea/controller/WebService.java

```diff
@@ -51,4 +56,17 @@ public Response getJobSpecification(@PathParam("id") int role_id)
             return Response.status(HttpStatus.INTERNAL_SERVER_ERROR_500, e.getMessage()).build();
         }
     }
+    @GET
+    @Path("/band-competency/{id}")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getBandCompetency(@PathParam("id") int role_id)
```

Comment:
you shouldn't take in role id when competencies are related to band, update to take in band id

Reply:
done